Makefile

```diff
@@ -2,6 +2,10 @@ all: clean package
 
 .PHONY: test
 test:
+	@echo ----- Running docker container for vault tests-----
+	docker run --cap-add=IPC_LOCK --rm -p 8200:8200 -d -e \
+		'VAULT_LOCAL_CONFIG={"backend": {"file": {"path": "/vault/file"}}, "listener":{"tcp":{"address":"0.0.0.0:8200","tls_disable":"true"}}}'\
+		vault server
```

Comment:
I believe we don't need this. We can definitely mock all these values in the test case itself. If you have written this to test the call to the vault server during the reveal, it's not required. It's like we are testing the vault, not the feature.